sdk/python/feast/repo_config.py

```diff
@@ -195,6 +195,12 @@ def __repr__(self) -> str:
         )
 
 
+def get_data_source_class_from_type(offline_store_type: str):
```

Comment:
Why is this method in `repo_config.py`?

Also, what happens when a data source can be used in two offline stores?

Reply:
Can you give an example of this? Shouldn't the source be tied to the offline store?

Reply:
You could in theory have an S3/parquet source that is readable by Redshift, Athena, or a File offline store.

Reply:
In theory we could push this problem down the road and generalize later, but I can see it potentially happening.

Reply:
Hmmm. Actually it turns out that the way i've implemented this, it doesn't have a hard dependency on the offline store type - it only needs the fully qualified class name for the data source class which we stuff into the  proto, and use to import the class dynamically